tests/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -2027,5 +2028,22 @@ public void TestAppendEmptyValue()
             Assert.Equal(13, df.Rows.Count);
             Assert.Equal(1, df.Columns[2].NullCount);
         }
+
+        [Fact]
+        public void TestApply()
+        {
+            int[] values = { 1, 2, 3, 4, 5 };
+            var col = new PrimitiveDataFrameColumn<int>("Ints", values);
+            var df = new DataFrame(col);
```

Comment:
I don't think `df` is used. Can it be removed?

Reply:
Yes, definitely.